examples/configs/gadget/MilliMillenium.conf

```diff
@@ -15,4 +15,4 @@ MinSnapshotIndex 8
 [Reader]
 SnapshotFormat gadget
 GroupFileFormat gadget2_long
-GroupParticleIdMask 0x00000003FFFFFFFF #only the least significant 34bits are the actual ids
\ No newline at end of file
+GroupParticleIdMask 0x00000003FFFFFFFF #only the least significant 34bits are the actual ids
```

Comment:
Are we going to have a config with a list of all possible options, or maybe you will put that somewhere else in the docs? If the former then can we add this option to the example.conf

Reply:
My plan was indeed to list all config options in the example.conf, in the docs themselves, or both. I certainly want to list all options in the future docs, but I think it would also be nice to have them spelt out in the [example config](https://github.com/SWIFTSIM/HBTplus/blob/organisation/examples/configs/Example.conf).

Reply:
For now, I have listed all possible options in the Example.conf ([ce07b5f](https://github.com/SWIFTSIM/HBTplus/pull/64/commits/ce07b5f195a0b69945c1edaa8335ac5b2f9c4ad9)). I provide an explanation within so that people understand what each parameter means. In the future, I will add those explanations to the official docs.